src/main/java/com/github/fge/jsonpatch/JsonPathParser.java

```diff
@@ -0,0 +1,14 @@
+package com.github.fge.jsonpatch;
+
+public class JsonPathParser {
+
+    public static String tmfStringToJsonPath(String path) {
+        if ("/".equals(path)) {
+            return "$";
+        }
+        final String jsonPath = "$" + path.replace('/', '.')
+                .replaceAll("\\.(\\d+)\\.", ".[$1].")
```

Comment:
Can we move those regexp to final statics fields and give them meaningful description?